examples/lb3-application/lb3app/test/acceptance.js

```diff
@@ -0,0 +1,130 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-lb3-application
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+const lb3App = require('../server/server');
+const request = require('@loopback/testlab').supertest;
+const assert = require('assert');
+const ExpressServer = require('../../dist/server').ExpressServer;
+require('should');
+
+let app;
+
+function json(verb, url) {
+  // use the original app's server
+  return request(app.server)
+    [verb](url)
+    .set('Content-Type', 'application/json')
+    .set('Accept', 'application/json')
+    .expect('Content-Type', /json/);
+}
+
+describe('LoopBack 3 style acceptance tests', function () {
+  before(async function () {
+    app = new ExpressServer();
+    await app.boot();
+    await app.start();
+  });
+
+  after(async () => {
+    await app.stop();
+  });
+
+  context('basic REST calls for LoopBack 3 application', () => {
+    it('creates and finds a CoffeeShop', function (done) {
+      json('post', '/api/CoffeeShops')
+        .send({
+          name: 'Coffee Shop',
+          city: 'Toronto',
+        })
+        .expect(200)
+        .end(function (err, res) {
+          assert(typeof res.body === 'object');
+          assert(res.body.name);
+          assert(res.body.city);
+          assert.equal(res.body.name, 'Coffee Shop');
+          assert.equal(res.body.city, 'Toronto');
+          done();
+        });
+    });
+
+    it("gets the CoffeeShop's status", function (done) {
+      json('get', '/api/CoffeeShops/status').expect(200, function (err, res) {
+        res.body.status.should.be.equalOneOf(
+          'We are open for business.',
+          'Sorry, we are closed. Open daily from 6am to 8pm.',
+        );
+        done();
+      });
+    });
+
+    it('gets external route in application', function (done) {
+      json('get', '/ping').expect(200, function (err, res) {
+        assert.equal(res.text, 'pong');
+        done();
+      });
+    });
+  });
+
+  context('authentication', () => {
+    let User;
+
+    before(() => {
+      User = lb3App.models.User;
+    });
+
+    it('creates a User and logs them in and out', function (done) {
+      User.create({email: 'new@email.com', password: 'L00pBack!'}, function (
+        err,
+        user,
+      ) {
+        assert.equal(user.email, 'new@email.com');
+        User.login(
```

Comment:
glad to see you got it work 👍 and nice to verify that User authentication works as expected.